src/components/popup-menu/popup-menu.scss

```diff
@@ -3,7 +3,7 @@ $sgPopupMenuPadding: rhythm(1 / 2);
 $sgPopupMenuTriangleBorder: rhythm(1 / 3);
 $sgPopupMenuHeight: rhythm(22 / 12); // 44px
 $sgPopupMenuBoxShadow: $defaultBottomShadow;
-$sgPopupMenuBorderRadiusTop: 8px;
+$sgPopupMenuBorderRadiusTop: $defaultBorderRadius;
```

Comment:
assigning variables to variables doesn't look so hot. I'd skip that, especially because we are not doing it everywhere.